service/metric/metrics.go

```diff
@@ -0,0 +1,50 @@
+package metric
```

Comment:
we use the same basic metrics in `snapstorage` and other store projects, so for consistency reasons, I left naming and labels the same.